docs/tutorials.rst

```diff
@@ -15,16 +15,32 @@
 Tutorials
 =========
 
+Example optimization problems
+-----------------------------
+
+Introductory
+^^^^^^^^^^^^
+
 .. toctree::
-   :caption: Introductory
    :maxdepth: 1
 
-   notebooks/introduction
+   notebooks/expected_improvement
    notebooks/thompson_sampling
 
+Advanced
+^^^^^^^^
+
 .. toctree::
-   :caption: Advanced
    :maxdepth: 1
 
    notebooks/inequality_constraints
    notebooks/failure_ego
+
+Trieste functionality in detail
+-------------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   Create a custom acquisition function <notebooks/failure_ego>
+   Create a custom model optimization routine <notebooks/failure_ego>
```

Comment:
neither me nor james could work out how to link to specific sections in a notebook